test/apps/iodemo/io_demo.cc

```diff
@@ -940,7 +941,7 @@ class P2pDemoCommon : public UcxContext {
         return (data_size + chunk_size - 1) / chunk_size;
     }
 
-    static void validate_failure(const UcxConnection *conn,
+    static void validate_failure(UcxConnection *conn,
```

Comment:
why need to change to non-const?
if it's because of set_verbose_log_prefix, maybe can refactor set_verbose_log_prefix to avoid this need

Reply:
as said in another conversation, `set_verbose_log_prefix() `needs to be called after ep established but we have no idea when is ep established especially for AM scenario. so `get_log_prefix() `will try to call `set_verbose_log_prefix() ` if `-V` specified but verbose log prefix not yet set.

Reply:
can we change log prefix when connection is established? i would expect data flow would not need to set log prefix.